i18n/__init__.py

```diff
@@ -45,8 +46,6 @@ def __call__(self, **kwargs):
         args = self.parser.parse_known_args(self.args)[0]
         for key, val in kwargs.items():
             setattr(args, key, val)
-        if args.config:
-            config.CONFIGURATION = config.Configuration(args.config)
-        else:
-            config.CONFIGURATION = config.Configuration(config.LOCALE_DIR.joinpath('config.yaml').normpath())
+        root_dir = kwargs.get('root_dir')
+        self.configuration = config.Configuration(filename=args.config, root_dir=root_dir)
```

Comment:
You can drop the kwargs.get, and instead use `root_dir=args.root_dir`, I think? Unless root_dir can be missing from kwargs?